components/ProductCard.vue

```diff
@@ -67,7 +67,10 @@ export default Vue.extend({
   },
   computed: {
     promoDescription(): string | undefined {
-      return this.card.potentialPromotions[0]?.description
+      return (
+        this.card.potentialPromotions[0] &&
+        this.card.potentialPromotions[0].description
+      )
```

Comment:
Why?

Reply:
@coopjakob had some problems with optional chaining in Storybook so I changed it. Will try to make it work before merging